ring_describer_test.go

```
@@ -344,6 +304,7 @@ func (c *trackingRingConnection) querySystem(context.Context, string, ...any) *I
 }
 func (c *trackingRingConnection) getIsSchemaV2() bool { return c.schemaV2 }
 func (*trackingRingConnection) setSchemaV2(bool)      {}
+func (*trackingRingConnection) isScyllaConn() bool { return false }
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)